### PR TITLE
Compiled the TrafficLayer component into lib

### PR DESCRIPTION
**Pull request by Michael-Brooks**
My company needed this feature pretty sharpish so I have gone through the trouble of running `npm run build:lib`.

This is my very first ReactJS package I have contributed to, so please let me know if that is the correct way to compile the code.

It's currently running very well on my machine, so thank you.

### CODE REVIEW

File: src/app/pages/basics/TrafficLayerExample.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* global google */
+import {
+  default as React,
```
Comment (batjko) [inline src/app/pages/basics/TrafficLayerExample.js]: I'm not sure this import notation should be encouraged. What's the reason you're not doing the normal 
`import React, { Component } from 'react'` here?

Comment (Michael-Brooks) [inline src/app/pages/basics/TrafficLayerExample.js]: This PR is meant to go on top of #417 but I'm not sure it has been done correctly. His original PR created the TrafficLayer, but he didn't compile it into the code so I took it that extra step in order to get it working correctly.

Comment (thzinc) [inline src/app/pages/basics/TrafficLayerExample.js]: I'll admit, I copied `HeatmapLayer` and changed the relevant bits for `TrafficLayer`. This is in `HeatmapLayer` too. I'm happy update them both if desired, but either way, I'd also love to get `TrafficLayer` into the package.

**Comment by tomchentw** [comment]
Released [6.2.0](https://github.com/tomchentw/react-google-maps/releases/tag/v6.2.0)

**Pull request closed without merging**